README.md

```diff
@@ -1,6 +1,10 @@
+# [DEPRECATED] Operator SDK Samples
+
+This repository has been deprecated. The sample operator projects have been moved to `testdata/` in [Operator SDK][https://github.com/operator-framework/operator-sdk/] repository. For more info see the [enhancement proposal][https://github.com/operator-framework/enhancements/blob/master/enhancements/samples-generation.md] for automatic sample generation using Go.  
```

Comment:
Need to fix these links. The second set of []s should be ()s.

Reply:
Agreed. If you want to use square brackets you have to use an alias at the bottom of the document. 

```
[link text][link-alias]
...
[link-alias](actual/url)
```

OR

Do what @jberkhahn said and just change the second set of square brackets to parentheses:
```
[enhancement proposal](https://github.com/operator-framework/enhancements/blob/master/enhancements/samples-generation.md)
```
Which will render like this:
[enhancement proposal](https://github.com/operator-framework/enhancements/blob/master/enhancements/samples-generation.md)

Reply:
ops. Typo issue.. Tks.